docs/leverage.md

```diff
@@ -0,0 +1,3 @@
+For shorts, the currency which pays the interest fee for the `borrowed` currency is purchased at the same time of the closing trade (This means that the amount purchased in short closing trades is greater than the amount sold in short opening trades).
```

Comment:
Maybe it's intentional as "internal" docs for now - but this way it won't be shown anywhere.
It's a standalone file that's not included (search docs for `plugins.md`) - nor is it in mkdocs.yml (so it get's it's own link in the doc page).

Considering that it's "for now" not an exposed part - i'm not sure what / where to go with this...

Reply:
I put `- Leverage: leverage.md` inside `mkdocs.yml`, right after backtesting

Reply:
not commited / not part of this PR?
It's not showing up in the list of changed files for this PR ...

Reply:
It was locally sorry, I should have just not said anything